fiftyone/core/collections.py

```
@@ -11412,12 +11573,15 @@
 
         return False
 
-    def _is_label_field(self, field_name, label_type_or_types):
+    def _is_label_field(self, field_name, label_type_or_types=None):
         try:
             label_type = self._get_label_field_type(field_name)
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)